Resources/public/js/models/ez-contentmodel.js

```diff
@@ -202,6 +202,48 @@ YUI.add('ez-contentmodel', function (Y) {
         copy: function (options, parentLocationId, callback) {
             options.api.getContentService().copyContent(this.get('id'), parentLocationId, callback);
         },
+
+        /**
+         * Gets content's locations list
+         *
+         * @method getLocations
+         * @param {Object} options
+         * @param {Object} options.api (required) the JS REST client instance
+         * @param {Function} callback
+         */
+        getLocations: function (options, callback) {
```

Comment:
should be called `loadLocations` to reflect the fact it's asynchronous and does not return anything.